lib/miq_automation_engine/service_models/miq_ae_service_service_ansible_tower.rb

```diff
@@ -5,5 +5,7 @@ class MiqAeServiceServiceAnsibleTower < MiqAeServiceService
 
     expose :launch_job
     expose :job
+    expose :job_options
+    expose :job_options=
```

Comment:
@bzwei 
Are the job_options going to be simple data types or could they be ServiceModel objects?